src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportIntegrationTest.java

```diff
@@ -29,6 +29,21 @@
 
 public final class GenomicsDBImportIntegrationTest extends CommandLineProgramTest {
 
+    private static final File TEST_REFERENCE_GENOME = new File(largeFileTestDir + "/Homo_sapiens_assembly38.20.21.fasta");
+
+    private static final String HG_00096 = largeFileTestDir + "gvcfs/HG00096.g.vcf.gz";
+    private static final String HG_00268 = largeFileTestDir + "gvcfs/HG00268.g.vcf.gz";
+    private static final String NA_19625 = largeFileTestDir + "gvcfs/NA19625.g.vcf.gz";
+    private static final List<String> LOCAL_GVCFS = Arrays.asList(HG_00096, HG_00268, NA_19625);
+
+    private static final String HG_00096_CLOUD = getGCPTestInputPath() + "large/gvcfs/HG00096.g.vcf.gz";
+    private static final String HG_00268_CLOUD = getGCPTestInputPath() + "large/gvcfs/HG00268.g.vcf.gz";
+    private static final String NA_19625_CLOUD = getGCPTestInputPath() + "large/gvcfs/NA19625.g.vcf.gz";
```

Comment:
I changed it to build these paths off of the local list at each test.  It's concise and can't fail statically.